corehq/apps/registration/templates/registration/base.html

```diff
@@ -2,6 +2,21 @@
 
 {% block head %}{{ block.super }}
     {% include 'analytics/fullstory.html' %}
+    <!-- Facebook Pixel Code -->
+    <script>
+    !function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function(){n.callMethod?
+    n.callMeyythod.apply(n,arguments):n.queue.push(arguments)};if(!f._fbq)f._fbq=n;
+    n.push=n;n.loaded=!0;n.version='2.0';n.queue=[];t=b.createElement(e);t.async=!0;
+    t.src=v;s=b.getElementsByTagName(e)[0];s.parentNode.insertBefore(t,s)}(window,
+    document,'script','https://connect.facebook.net/en_US/fbevents.js');
+    fbq('init', '301081893674207');
```

Comment:
Is this a user id / password? If so, it should probably go into a settings file.
We also shouldn't be including this on enikshay / icds / other enterprise sites, right?

Reply:
yep. I just copied the code from an email. I'll go ahead and update. good catch

Reply:
should probably get a new ID now that this will be in our git history.